README.md

```diff
@@ -284,7 +284,7 @@ SPARK|2.1.3, 2.3.2|APACHE_CASSANDRA
 KAFKA|2.7.1, 2.8.1, 3.0.0|
 KAFKA_REST_PROXY|5.0.0|KAFKA
 KAFKA_SCHEMA_REGISTRY|5.0.0, 5.0.4|KAFKA
-OPENSEARCH|1.0.0
+OPENSEARCH|1.2.4
```

Comment:
a doco change for OpenSearch team's [INS-17267](https://instaclustr.atlassian.net/browse/INS-17267) - Long Hoang